docs/develop/config.md

```diff
@@ -36,22 +36,60 @@ That's it configuration-wise. If we try to create a record through the API, your
 Did the changes work? We are going to try to create a new record:
 
 ``` console
-$ curl -k -XPOST -H "Content-Type: application/json" https://localhost:5000/api/records/ -d '
-{
-    "access": {
-        "metadata_restricted": "false",
-        "files_restricted": "false"
+$ curl -k -XPOST -H "Content-Type: application/json" https://localhost:5000/api/records/ -d '{
+	"_access": {
+		"metadata_restricted": "false",
+		"files_restricted": "false"
```

Comment:
Booleans should not be quoted, right? Should be `"metadata_restricted": false` and so on.

Reply:
True, thanks!